templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -924,3 +928,7 @@ contents:
       # Remove bridges created by ovn-kubernetes
       ovs-vsctl --timeout=30 --if-exists del-br br-int -- --if-exists del-br br-local
     fi
+
+    # we need to notify to systemd that we are ready as this is setup as a
+    # notify unit type 
+    systemd-notify --ready || true
```

Comment:
Should this be in handle_exit so we always run it even if the script exits early for some reason?

Reply:
done